backend/lib/backend_web/controllers/comment_controller.ex

```diff
@@ -0,0 +1,50 @@
+defmodule BackendWeb.CommentController do
+  use BackendWeb, :controller
+
+  alias Backend.Comments
+  alias Backend.Comments.Comment
+
+  action_fallback BackendWeb.FallbackController
```

Comment:
Add userAuth plug here.  The actions that matter I believe are just create, update, and delete

Reply:
Should likely also include, update, and delete

Reply:
Even if we don't use them